### DISCUSSION TITLE
Accepted Answers VS Discussions Answered?

### DISCUSSION THREAD

**Discussion by justin-2028** [community/community#19020] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/96811261/174728456-4e897ff7-5fbb-4867-8bef-600bd6dcf51f.png)

For the Galaxy Brain Badge, you need 2 accepted answers. So far I have 4 of my discussion responses marked as the answer, hence the "highlight", but I still don't have the badge. I have responded to 4 different people and all on public repos, so I'm confused on why I haven't unlocked it yet. What could it be?

**Comment by TharinduThilakshana** [comment]
historic answers not getting for badge, do 2 new answers.

**Reply by justin-2028** [reply]
2 of the answers were within the last 24 hours, so that's why I'm confused

**Reply by SenseiRofu** [reply]
m8 you must to check this link https://github.com/Schweinepriester/github-profile-achievements

**Reply by Son-Nguyen48** [reply]
> m8 you must to check this link https://github.com/Schweinepriester/github-profile-achievements
well thank you. Have a nice day!

**Reply by king4arabs** [reply]
Perfect and thanks

**Reply by aungkohtat** [reply]
Thanks a lot

**Comment by thomscoder** [comment, marked as answer]
Hi @justin-2028 
It is possibly a bug. I've looked into your contribution activity and they look fine. You met the criteria. It is not the first time I've seen this and the other profile that had the same problem was because he/she replied to him/herself. I don't know if that is your case, but keep in mind that answering your own questions might be a violation and in certain cases can prevent you from getting the badge.

If you didn't I think it is a bug

**Reply by justin-2028** [reply]
Thanks for the elaboration, I might try and contact GitHub about it

**Reply by thomscoder** [reply]
Yes
You can also open an issue.
Best of luck!

**Reply by SenseiRofu** [reply]
m8 you must to check this link https://github.com/Schweinepriester/github-profile-achievements

**Reply by SideeID** [reply]
m8 you must to check this link https://github.com/Schweinepriester/github-profile-achievements

**Comment by kulasekara02** [comment]
how can I open a issue

**Reply by king4arabs** [reply]
Create new fork and add comment

**Reply by kulasekara02** [reply]
Thanks brother

**Comment by Pathum-lakshan** [comment]
historic answers not getting for badge, do 2 new answers.

**Reply by king4arabs** [reply]
It is ok ,, that happened sometimes

**Comment by king4arabs** [comment]
Great

**Reply by SideeID** [reply]
Great

**Reply by usman-302** [reply]
they why all they annoying popups saying it should be marked answered ?

**Comment by usman-302** [comment]
they why all they annoying popups saying it should be marked answered ?

**Reply by usman-302** [reply]
reply

**Comment by usman-302** [comment]
i want to heir a person for job

**Comment by usman-302** [comment]
> Greathow can I open a issue

**Comment by usman-302** [comment]
how can I open a issue

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @justin-2028. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Greetlinj** [comment]
Thanks for sharing this comparison! It would be useful to clarify how accepted answers affect search visibility and future readers. For example, if an answer is marked as accepted but another solution becomes better later, should it be updated or re‑marked? Also, how does this impact contributors who helped with alternate solutions?
